dkist/io/asdf/resources/schemas/varying_celestial_transform-1.0.0.yaml

```diff
@@ -30,7 +30,7 @@ properties:
       - tag: "core/ndarray-1.0.0"
       - tag: "tag:stsci.edu:asdf/unit/quantity-1.1.0"
   projection:
-    $ref: "tag:stsci.edu:asdf/transform/transform-1.2.0"
+    $ref: "http://stsci.edu/schemas/asdf/transform/transform-1.2.0"
```

Comment:
Can this be / should this be `asdf://`?

Reply:
Unfortunately this is still `http`:
https://github.com/asdf-format/asdf-transform-schemas/blob/82dd76605ee62c85cf9945feea3028924a49dc3d/resources/stsci.edu/schemas/transform-1.2.0.yaml#L4